dev/reference/tools/mydumper.md

```diff
@@ -35,17 +37,123 @@ It can be [downloaded](/dev/reference/tools/download.md) as part of the Enterpri
 
 ### Usage example
 
-Command line parameter:
+To back up data from TiDB, add command line parameters to the following command according to your situation:
 
-```
+{{< copyable "shell-regular" >}}
+
+```bash
 ./bin/mydumper -h 127.0.0.1 -u root -P 4000
 ```
 
 ## FAQ
 
-### Is the source code for these changes available?
+### How to determine if the Mydumper I am using is the PingCAP optimized version?
+
+Execute the following command：
+
+{{< copyable "shell-regular" >}}
+
+```bash
+./bin/mydumper -V
+```
+
+If the output contains `git_hash` (`d3e6fec8b069daee772d0dbaa47579f67a5947e7` in the following example), you are using the PingCAP optimized version of Mydumper:
+
+```
+mydumper 0.9.5 (d3e6fec8b069daee772d0dbaa47579f67a5947e7), built against MySQL 5.7.24
+```
+
+### What can I do if an "invalid mydumper files for there are no `-schema-create.sql` files found" error is reported when using Loader to restore the data backed up by Mydumper?
+
+Check whether the `-T` or `-tables-list` option is used when using Mydumper to back up data. If these options are used, Mydumper does not generate the `-schema-create.sql` file.
+
+**Solution**: Create the `{schema-name}-schema-create.sql` file in the directory for data backup of Mydumper. Write "CREATE DATABASE `{schema-name}`" to the file, and then run Loader.
+
+### Why is the TIMESTAMP type data exported using Mydumper inconsistent with that in the database?
+
+Check whether the time zone of the server running Mydumper is consistent with that of the database. Mydumper converts the TIMESTAMP type data according to the time zone of the server. You can add the `--skip-tz-utc` option to disable the conversion of dates and times.
+
+### How to configure the `-F,--chunk-filesize` option of Mydumper?
+
+Mydumper divides the data of each table into multiple `chunk` according to the value of this option during backup. Each `chunk` is saved to a file with a size of about `chunk-filesize`. In this way, the data is divided into multiple files so that the import speed is improved by using the parallel processing of Loader/TiDB Lightning. If you later use **Loader** to restore the backup files, it is recommended to set the value of this option to `64` (in MB); If you use **TiDB Lightning** to restore files, `256` (in MB) is recommended.
+
+### How to configure the `-s --statement-size` option of Mydumper?
+
+Mydumper uses this option to control the size of `Insert Statement` which defaults to `10000000` (about 1 MB). Use this option to avoid the following errors occurring when restoring data:
```

Comment:
```suggestion
Mydumper uses this option to control the size of `Insert Statement` which defaults to `10000000` (about 1 MB). Use this option to avoid the following error when restoring data:
```